xml/sle_update_preparation.xml

```diff
@@ -345,9 +345,9 @@
    <remark>toms 2016-07-27: from bsc#970153, c#24</remark>
    <remark>jufa 2021-02-09: FIXME check if still relevant, check which versions are affected</remark>
    <para>
-    During the update from SP1 to SP2, MD5-based certificates were disabled as
-    part of a security fix. If you have certificates created as MD5, re-create
-    your certificates with the following steps:
+    During the update from &slea; 11 SP1 to SP2, MD5-based certificates are
+    disabled as part of a security fix. If you have certificates created as MD5,
```

Comment:
It would be more accurate to say that they are "no longer supported" rather than being "disabled". The correction only makes the misunderstanding worse.
Also, "certificates based on MD5 hashes" should be better phrasing than "certificates created as MD5".